src/store/index.js

```diff
@@ -0,0 +1,109 @@
+import { createStore } from "vuex";
+import axios from "axios";
+import router from "../router";
+
+const store = createStore({
+  state: {
+    posts: [],
+    showDeleteAlert: false,
+    post: {
```

Comment:
This object shouldn't be in the store. It should be handled individually in creating posts and updating posts components.